src/services/textChanges.ts

```diff
@@ -345,6 +405,13 @@ namespace ts.textChanges {
             this.replaceRangeWithNodes(sourceFile, getAdjustedRange(sourceFile, startNode, endNode, options), newNodes, options);
         }
 
+        public replaceAndDeleteNodeWithNodes(sourceFile: SourceFile, oldNode: Node, newNodes: readonly Node[], deleteNodes: Node[], replaceOptions: ChangeNodeOptions = useNonAdjustedPositions, deleteOptions: ConfigurableStartEnd = { leadingTriviaOption: LeadingTriviaOption.IncludeAll }): void {

```

Comment:
This name is a bit confusing - I probably wouldn't even extract this to a function. It's pretty simple and it looks like you only use it once anyway.

Reply:
Agree, done.